react-client-app/src/App.jsx

```diff
@@ -14,6 +14,7 @@ import ScanReportAdminForm from './views/ScanReportAdminForm'
 import Home from './components/Home';
 import { getScanReportConcepts, m_allowed_tables, useDelete, useGet, usePost, mapConceptToOmopField, saveMappingRules } from './api/values'
 import UploadScanReport from './components/UploadScanReport'
+import DatasetsContent from './components/DatasetsContent'
```

Comment:
I would put `DatasetsContent.jsx` in `./views` now with the scan report admin form view. This isn't really a "component" seeing as is essentially the whole page. The same is true for other "components" in the `components` folder. But we can change them later. For this PR, just move `DatasetsContent`.